### DISCUSSION TITLE
Lost connection to GitHub reop

### DISCUSSION THREAD

**Discussion by Shivom-HikeMedical** [vercel/community#1553] [Help] [answered]
### Summary

Vercel is not connected to the git hub repo and the latest merges are not showing up on deployment. Vercel says there is "an unexpected internal error"

**Comment by sinuheshinbr** [comment]
Having the same issue here, any updates on this?

**Comment by amyegan** [comment, marked as answer]
I'm not sure what happened to disconnect your repo, since there are many possibilities, but you should be able to re-connect it. You may need to reconfigure the Vercel for GitHub integration and  to get it working again.

In your Project settings, go to the Git tab. You should have your GitHub repo in the Connected Git Repository section. If you see buttons to connect to GitHub, GitLab, or Bitbucket, then your Project needs to be re-connected to the repo. 
<img src="https://user-images.githubusercontent.com/8247105/219490760-97f70091-aa64-4f1c-b69a-317227b5470d.png" alt="Settings page with git connection buttons" />

If the connection is failing, you may have a permissions issue. Vercel should appear in the [applications](https://github.com/settings/installations) section of your GitHub settings. Make sure it has access to the repository and try connecting again

**Comment by baukevanderlaan** [comment]
Having the same issue, but only with repositories on Bitbucket. The `Connected Git Repository` shows an error and when clicking `Reconnect` a loading animations shows up that never stops. Browser console says `GET https://vercel.com/api/v1/integrations/bitbucket/verify 500`. `Login Connections` in `Personal Account Settings` gives the same never ending loading animation.

<img width="970" alt="Error in Connected Git Repository" src="https://user-images.githubusercontent.com/31567440/221406288-34275383-f38c-497e-a6e5-92879dbf1530.png">

<img width="905" alt="Loading animation in Connected Git Repository" src="https://user-images.githubusercontent.com/31567440/221406304-5532a3e7-c87b-42c0-9c29-b9093714c93d.png">

<img width="962" alt="Loading animation in Login Connections" src="https://user-images.githubusercontent.com/31567440/221406317-58b92479-7e31-4c2b-ae28-62f3859e45ad.png">

**Reply by amyegan** [reply]
Are you still seeing this error? It looks like [Bitbucket recently had an incident affecting APIs](https://bitbucket.status.atlassian.com/incidents/cd10cjhvtjfr) that is resolved now

**Reply by baukevanderlaan** [reply]
Thanks, but unfortunately I‘m still seeing the same error.
